group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorServiceTest.java

```diff
@@ -595,4 +599,98 @@ public void testHeartbeatCoordinatorException() throws Exception {
             future.get()
         );
     }
+
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testFetchOffsetsWithoutWithoutRequireStable(
```

Comment:
nit: Were these tests named before we parameterized them? This one and the one below say Without (Without) Require stable, but looks like we test both.

Reply:
Good catch. You're right.